releasenotes/notes/process-agent-add-scrubber-and-ctr-source-env-vars-3543c08590eb245d.yaml

```diff
@@ -1,6 +1,7 @@
+deprecations:
+  - Remove `DD_PROCESS_AGENT_CONTAINER_SOURCE` environment variable from the process agent. The list of container sources now entirely depends on the activated features.
```

Comment:
```suggestion
  - Removes the `DD_PROCESS_AGENT_CONTAINER_SOURCE` environment variable from the Process Agent. The list of container sources now entirely depends on the activated features.
```

Reply:
Thanks for the suggestion, @alai97 ! fixed!